crypto/evp/e_aes.c

```diff
@@ -4277,10 +4302,10 @@ static int aes_ocb_cleanup(EVP_CIPHER_CTX *c)
     return 1;
 }
 
-BLOCK_CIPHER_custom(NID_aes, 128, 16, 12, ocb, OCB,
```

Comment:
do we not need the 'BLOCK_CIPHER_custom' function any more?

Reply:
Yes we still do. It's used for AES in gcm mode, which we support. Here we are switching  to BLOCK_CIPHER_custom_fips to prevent AES from being used in ocb mode, which we are disallowing under FIPS.

Note the "ocb" parameter to this function - that's telling the macro expansion what cipher mode we're defining here.